MediaManager/Plugin.MediaManager/CrossMediaManager.cs

```diff
@@ -17,25 +18,27 @@ public static IMediaManager Current
     {
       get
       {
-        var ret = Implementation.Value;
-        if (ret == null)
+      
+        if (Implementation.Value == null)
         {
           throw NotImplementedInReferenceAssembly();
         }
-        return ret;
+        return Implementation.Value;
       }
     }
 
     static IMediaManager CreateMediaManager()
     {
 #if PORTABLE
+        Debug.WriteLine("PORTABLE Reached");

```

Comment:
If you want those debugging-lines to remain in the code, the message should be a bit more descriptive, that it also is valuable outside of the scope of this library.